build-info/github.com/spring-projects/spring-boot/build.yaml

```diff
@@ -0,0 +1,5 @@
+---
+preBuildScript: |
+    sed -i -e 's/.*id "io.spring.gradle-enterprise-conventions.*//' -e 's/include "spring-boot-project:spring-boot-docs"//' settings.gradle
```

Comment:
This will be partially solved by https://github.com/project-ncl/gradle-manipulator/pull/443